pkg/qiniu/client.go

```diff
@@ -43,23 +43,32 @@ type Config struct {
 	Domain string `json:"domain"`
 }
 
-// Client for the operation with qiniu cloud
-type Client struct {
+type Client interface {
```

Comment:
suggest to add comments for this exported interface

Reply:
added~